src/clusterfuzz/fuzz/__main__.py

```diff
@@ -31,7 +30,7 @@ def main():
       '-e',
       '--engine',
       help='Fuzzing engine.',
-      choices=['libFuzzer', 'honggfuzz'],
+      choices=['libFuzzer', 'afl', 'honggfuzz'],
```

Comment:
can we add this somewhere in constants and use in both places OR best to derive it from some fuzzing engines integrated in cf.

Reply:
Will do this in another PR to keep this one clean.